h2o-core/src/main/java/water/util/Log.java

```diff
@@ -334,12 +334,9 @@ private static void setLog4jProperties(String logDir, java.util.Properties p) th
     if( _logger != null ) return _logger; // Test again under lock
 
     boolean launchedWithHadoopJar = H2O.ARGS.launchedWithHadoopJar();
-    String log4jConfiguration = System.getProperty ("log4j.configuration");
     String h2oLog4jConfiguration = System.getProperty ("h2o.log4j.configuration");
```

Comment:
This will still cause the problem. When user specifies the custom `h2o.log4j.configuration` it is expected that the user will correctly specify the path to log files on so on so it's compatible with our implementation. However the log names are dynamically created and contain also IP and port of the current node - therefore there's a big chance that this will cause problems as well

Reply:
I think it is fine because users don't usually customize the log configuration. At least not the Hadoop users. Whatever was working before is fine for now :)